envoy/server/admin.h

```diff
@@ -102,7 +119,7 @@ class Admin {
      */
     virtual bool nextChunk(Buffer::Instance& response) PURE;
   };
-  using HandlerPtr = std::unique_ptr<Handler>;
+  using RequestPtr = std::unique_ptr<Request>;
```

Comment:
Why are we renaming these classes? I read the entire source code and feel there is no downside to keeping the original name.

Reply:
In its current state the turn 'Handler' was overloaded (and that's been true for only the most recent PR in this series).

See the above taxonomy: a Handler can generate Requests, and has a lifetime matching the Server. A Request object is created for each http request.